### ISSUE TITLE
Permission denied for fusionauth-app.gc.log in docker container

### ISSUE THREAD

**Issue by pigletto** [FusionAuth/fusionauth-issues#2392]
### Description 
When using docker image fusionauth/fusionauth-app:1.47.0 with docker-compose it is not possible to start container due to the following error:

[0.000s][error][logging] Error opening log file '/usr/local/fusionauth/logs/fusionauth-app.gc.log': Permission denied

### Observed in 
1.47.0

### Affects versions
1.47.0

Introduced via :
- https://github.com/FusionAuth/fusionauth-issues/issues/2388

### Steps to reproduce
Just try to use fusionauth/fusionauth-app:1.47.0 image

### Expected behavior
Container starts properly

### Platform
 - Device: Desktop
 - OS: Linux Manjaro + docker compose

### Related
- https://github.com/FusionAuth/fusionauth-issues/issues/2388
- https://github.com/FusionAuth/fusionauth-issues/issues/2391

**Comment by patricknwn** [comment]
Same here, running latest on Google Kubernetes. It will not start, rendering my authenticated area inaccessible!

```
2023-07-27 17:51:45.389 HKT fusionauth Error: A fatal exception has occurred. Program will exit.
2023-07-27 17:51:45.389 HKT fusionauth Error: Could not create the Java Virtual Machine.
2023-07-27 17:51:45.389 HKT fusionauth Invalid -Xlog option '-Xlog:gc:file=/usr/local/fusionauth/logs/fusionauth-app.gc.log:time:filecount=10,filesize=1M', see error log for details.
2023-07-27 17:51:45.389 HKT fusionauth [0.000s][error][logging] Initialization of output 'file=/usr/local/fusionauth/logs/fusionauth-app.gc.log' using options 'filecount=10,filesize=1M' failed.
2023-07-27 17:51:45.388 HKT fusionauth [0.000s][error][logging] Error opening log file '/usr/local/fusionauth/logs/fusionauth-app.gc.log': Permission denied
2023-07-27 17:51:45.383 HKT fusionauth --> Logging to /usr/local/fusionauth/logs/fusionauth-app.log
```

**Comment by pigletto** [comment]
Workaround is to create own docker image using the Dockerfile like:

ARG VERSION=latest
FROM fusionauth/fusionauth-app:$VERSION
USER root
RUN touch /usr/local/fusionauth/logs/fusionauth-app.gc.log && chown fusionauth:fusionauth /usr/local/fusionauth/logs/fusionauth-app.gc.log
USER fusionauth

**Comment by patricknwn** [comment]
I reverted to 1.46.0 for now and will be hesitant to follow the latest in the future for sure. This should have been a failed build and not pushed to the artifact repositories!

**Comment by Aaron-Ritter** [comment]
Same problem for me, and it is related to this #2388 , I'd argue it should not log in to the pod unless specified.

```
Defaulted container "fusionauth" out of: fusionauth, wait-for-db (init), wait-for-search (init)
Starting fusionauth-app...
  --> Logging to /usr/local/fusionauth/logs/fusionauth-app.log
[0.000s][error][logging] Error opening log file '/usr/local/fusionauth/logs/fusionauth-app.gc.log': Permission denied
[0.000s][error][logging] Initialization of output 'file=/usr/local/fusionauth/logs/fusionauth-app.gc.log' using options 'filecount=10,filesize=1M' failed.
Invalid -Xlog option '-Xlog:gc:file=/usr/local/fusionauth/logs/fusionauth-app.gc.log:time:filecount=10,filesize=1M', see error log for details.
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
```

**Comment by robotdan** [comment]
In general, we always do recommend testing before you pull the latest tag. 

However, this is certainly and error on our side, and the image should not have shipped with this error. I apologize. An updated image will be available shortly.

**Comment by robotdan** [comment]
Updated docker image will be available in `1.47.1`.
